src/test/e2e/no-updated-extensions.ts

```diff
@@ -11,8 +11,8 @@ describe('No updated extensions', () => {
   const fileSystem = mock(FileSystem);
 
   const lastRecordedVersions = {
-    ID_1: '1.0.0',
-    ID_2: '1.0.0'
+    extensionId1: '1.0.0',
+    extensionId2: '1.0.0'
```

Comment:
Wondering why this needs to be changed?

Reply:
Because `ID_1` does not confirm to object/property naming conventions.

Reply:
I see, thanks @jerone, I think I found the relevant rule. Could we use `camelcase: ["error", {properties: "never"}]`?

https://eslint.org/docs/latest/rules/camelcase#properties-never

Reply:
Are you sure? 
This will disable all naming conventions for properties in the whole project, just for a test.

Reply:
Haha, I thought about that too. We can have another eslint config file under the test directory to add more exceptions for only tests, but then thought this project has been without this rule for long anyway and wondered if we need it 😛 

But I'm fine with a separate config for test if you prefer!